src/main/java/me/zeroeightsix/fiber/api/builder/ConfigTreeBuilder.java

```diff
@@ -25,20 +27,10 @@
  * <p> Usage example:
  * <pre>{@code
  * ConfigBranch config = ConfigTree.builder()
- *         .beginValue("version", "1.0.0")
- *             .withFinality()
- *             .beginConstraints() // checks the default value
- *                 .regex("\\d+\\.\\d+\\.\\d+")
- *             .finishConstraints()
- *         .finishValue()
+ *         .withValue("A", ConfigTypes.INTEGER, 10)
  *         .fork("child")
- *             .beginValue("A", 10)
- *                 .beginConstraints()
- *                     .composite(CompositeType.OR)
- *                         .atLeast(3)
- *                         .atMost(0)
- *                     .finishComposite()
- *                 .finishConstraints()
+ *             .beginValue("drops", ConfigTypes.STRING_SET, new HashSet<>(Arrays.asList("diamond", "cactus"))
```

Comment:
`ConfigTypes.STRING_SET` does not exist.

Reply:
Good catch.